OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/AuthenticatedWebView.java

```diff
@@ -136,6 +136,14 @@ public void onReceivedHttpError(WebView view, WebResourceRequest request,
                 super.onReceivedHttpError(view, request, errorResponse);
             }
 
+            @Override
+            public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request) {
+                if(request.getUrl().toString().contains("logout") || view.getUrl().contains("delete")) {
+                    return true;
+                }
+                return super.shouldOverrideUrlLoading(view, request);
+            }
+
```

Comment:
need to override the `shouldOverrideUrlLoading(WebView view, String url)` for the android 6.0 and below. otherwise, navigation is not working for android 6.0 and below.

Reply:
we can use the following

``` java
            /**
             * Method is usable for Android 6.0 and below otherwise app didn't get the control when a URL is
             * about to be loaded in the current WebView.
             * Ref: https://developer.android.com/reference/android/webkit/WebViewClient#shouldOverrideUrlLoading(android.webkit.WebView,%20java.lang.String)
             */
            @Deprecated
            @Override
            public boolean shouldOverrideUrlLoading(WebView view, String url) {
                if (url.contains("logout") || view.getUrl().contains("delete")) {
                    return true;
                }
                return super.shouldOverrideUrlLoading(view, url);
            }
```